compiler/passes/docs.cpp

```diff
@@ -180,25 +181,25 @@ void printModule(std::ofstream *file, ModuleSymbol *mod, unsigned int tabs, std:
   if (!mod->noDocGen()) {
     mod->printDocs(file, tabs, parentName);
 
-    Vec<VarSymbol*> configs = mod->getTopLevelConfigVars();
+    std::vector<VarSymbol*> configs = mod->getTopLevelConfigVars();
     if (fDocsAlphabetize)
-      qsort(configs.v, configs.n, sizeof(configs.v[0]), compareNames);
-    forv_Vec(VarSymbol, var, configs) {
+      std::sort(configs.begin(), configs.end(), compareNames);
```

Comment:
I don't think C++ std::sort supports passing a C function pointer. I think it's reasonable to continue to call qsort on the std::vector (the C++ standard promises this is OK). If you want to use std::sort, AFAIK you need to define a function object with a method returning bool - for <.

Reply:
Hmm, you're right.  Guess I didn't read that carefully enough.  I'll update it, thanks!